packages/transport/src/modules/devices/backbone/DeviceClient.ts

```diff
@@ -8,4 +9,8 @@ export class DeviceClient extends RESTClient {
     public async createDevice(value: BackbonePostDevicesRequest): Promise<ClientResult<BackbonePostDevicesResponse>> {
         return await this.post<BackbonePostDevicesResponse>("/api/v1/Devices", value, {});
     }
+
+    public async setCommunicationLanguage(value: BackboneSetCommunicationLanguageRequest): Promise<ClientResult<void>> {
+        return await this.put<void>("/api/v1/Devices/Self", value);
+    }
```

Comment:
we have two options

1. rename this function to updateCurrentDevice
2. update the value to just accept one string and make `{ "communicationLanguage": value }` the payload

The version we have here is destroying the REST client that tries to be complete.

Reply:
I'll take option 2 - or are there other updating operations planned?

Reply:
I'm not sure what this route currently accepts. But I thought is's more than the communicationLanguage

Reply:
According to swagger, it's only the language

Reply:
Ok, I'd still rename the function to be ready for future changes.